Makefile

```diff
@@ -2,20 +2,24 @@ NAME := libcs50
 VERSION := 8.0.2
 UPSTREAM := $(NAME)-$(VERSION)
 
+# installation directory (/usr by default)
+DESTDIR ?= /usr
+
 .PHONY: build
 build: clean
 	gcc -c -fPIC -std=gnu99 -Wall -o cs50.o src/cs50.c
```

Comment:
Supported per ec6d95f! Falls back to `cc` per https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html!